### ISSUE TITLE
Not creator of tool

### ISSUE THREAD

**Issue by Johnthen1** [roro08/Sketchfab-Ripper#20]
I have done some digging and figured out that roro08 did not make this tool. that is why none of the issues are being solved! the server originally in the readme file is broken but I have found the server [https://discord.gg/C8nnf5kvhv] original server.
![image](https://user-images.githubusercontent.com/105382396/167913255-fccf2347-170f-486d-b77f-dd0a25dff1bc.png)

**Comment by Korozion** [comment]
Nice finding ! They talk about this issue on the Discord server :
in SketchfabRipper folder, open SketchfabRipper.ini with notepad, and replace the line beginning with
`sfConfig= ...`
with :
`sfConfig=wgmet7yBv1pVw9b1ZoNns2Yt8G4WNQ6xdsqyWUPbHwLgL0yqbre0lS4EdRb0R4Ud6fHYwHlxM9pEI6z50yUfay66kmvI5E7WvedLi2+wfI59LjyOBye7SGEafPgL6mfy`
That did the trick for me.

**Comment by Johnthen1** [comment]
yep, I hope people join the real server instead of this"personal one"

**Comment by Kylie-Wes** [comment]
Can confirm this. The real fixes are actually in the discord mentioned. Sad this person can't even link the right discord for a tool that isn't his.

**Comment by Lucien950** [comment]
Where is this discord? Can you please send?
